tests/common/strings.py

```diff
@@ -0,0 +1,6 @@
+"""
```

Comment:
Can we rename this to constants?

Reply:
changed its name from "strings.py" to "values.py"